### PR TITLE
feat(InviteMore): Relocate invite prompt for mobile friendliness.

### PR DESCRIPTION
**Pull request by muscat1**

### CODE REVIEW

**Comment by muscat1** [comment]
The intent was to move the 'invite more people' prompt to the bottom of the screen and tie its display/resize behaviour to the toolbar's.

Desktop:
Screenshot 2021-03-18 at 10 51 23

Mobile:
Screenshot 2021-03-18 at 10 51 59

File: react/features/conference/components/web/InviteMore.js [hunk 1]
```diff
@@ -74,7 +81,8 @@ function mapStateToProps(state) {
     const hide = interfaceConfig.HIDE_INVITE_MORE_HEADER;
 
     return {
-        _visible: isToolboxVisible(state) && isButtonEnabled('invite', state) && isAlone && !hide
+        shouldShow: isButtonEnabled('invite', state) && isAlone && !hide,
```
Comment (saghul) [inline react/features/conference/components/web/InviteMore.js:84]: Please prefix props coming from mapStateToProps with an underscore.

File: react/features/conference/components/web/InviteMore.js [hunk 2]
```diff
@@ -74,7 +81,8 @@ function mapStateToProps(state) {
...
+        toolboxVisible: isToolboxVisible(state)
```
Comment (saghul) [inline react/features/conference/components/web/InviteMore.js:85]: Is there a case when this is false and the above is true? IIRC we made it so the toolbar is always displayed when you are alone.

File: react/features/conference/components/web/Subject.js [hunk 3]
```diff
@@ -96,7 +96,7 @@ function _mapStateToProps(state) {
         _showParticipantCount: participantCount > 2 && !hideParticipantsStats,
         _showSubject: !hideConferenceSubject,
         _subject: getConferenceName(state),
-        _visible: isToolboxVisible(state) && participantCount > 1
```
Comment (saghul) [inline react/features/conference/components/web/Subject.js:99]: This change seems unrelated.

**Review by saghul — CHANGES_REQUESTED** [review]
Left some comments, PTAL.

File: react/features/conference/components/web/InviteMore.js (hunk 2 above)
Comment (muscat1) [reply react/features/conference/components/web/InviteMore.js:85]: From my testing, the toolbar still has the docking behaviour even when alone, but in this situation, I need to know whether the toolbar is displayed or not, separately, in order to add a class in the `render()` method.

File: react/features/conference/components/web/Subject.js (hunk 3 above)
Comment (muscat1) [reply react/features/conference/components/web/Subject.js:99]: Not necessarily, this PR enables the subject to be visible at the same time as the `InviteMore` component so we need this in order for that to happen.

**Comment by vp8x8** [comment]
This looks great!

File: react/features/conference/components/web/Subject.js (hunk 3 above)
Comment (saghul) [reply react/features/conference/components/web/Subject.js:99]: Ack

**Review by saghul — APPROVED** [review]
LGTM if vlad likes the CSS.
